meta-schema.graphql

```diff
@@ -0,0 +1,115 @@
+# Base types
+type Party {
+	id:ID!
+	idType:String! #PartyId, IPI, ISNI, MBID
+	displayName:String!
+	firstName: String
+	lastName: String
+}
+
+type Organisation {
+	id:String!
+	idType:String! #DPid, IPI, MBID
+	displayName:String!
+	formalName:String
+}
+
+# Music Types
+type TerritoryAgreement {
+	territoryCode:Int! #CISAC Territory standard
+	condition:String! # CWR inclusion/exclusion indicator
+}
+
+type InterestedPartyAgreement {
+	agreementRoleCode:String!
+	interestedPartyNumber:Int!
+	ipi:Int
+	ipiBase:Int
```

Comment:
What's the difference between `ipi` and `ipiBase`?

Reply:
"ipi: The IP Name Number is the code for a name or pseudonym related to an entity (natural person or a legal entity). One entity can have several names. Prince for example has the IPI-codes 00045620792 (Nelson Prince Rogers), 00052210040 (Prince) and 00334284961 (Nelson Prince R)
ipiBase: The IPI Base Number is the code for an entity itself. It has the pattern H-NNNNNNNNN-C."